packages/fluxible/utils/callAction.js

```diff
@@ -13,7 +13,8 @@ require('setimmediate');
  * If done callback supplied, that indicates non-Promise invocation expectation,
  * otherwise, Promise invocation.
  */
-function callAction (actionContext, action, payload, done) {
+function callAction (context, actionContext, action, payload, done) {
```

Comment:
@Vijar Just curious, is `context` added for accessing `_enableDebug` only, or are you planning to make more use of it later?